src/DotNetEnv/Parsers.cs

```diff
@@ -215,7 +215,7 @@ private static string ToUtf32Char (string hex)
                 .AtLeastOnce()
                 .Select(strs => new ValueActual(strs))
                 .Many()
-                .Select(vs => new ValueCalculator(vs));
+                .Select(vs => new ValueCalculator(vs.ToArray<IValue>()));
```

Comment:
Why is this getting added?

Reply:
Willbe solved with reverting that: https://github.com/tonerdo/dotnet-env/pull/93/files/e1e5f17d1f16eb4761136bffe777cade0240bdd5#r1510010054